__tests__/app.test.js

```diff
@@ -22,3 +23,32 @@ describe("GET /api", () => {
       });
   });
 });
+
+describe("GET /api/topics", () => {
+  test("200: Responds with an array of topic objects", () => {
+    const topics = [
+      {
+        description: "The man, the Mitch, the legend",
+        slug: "mitch",
+      },
+      {
+        description: "Not dogs",
+        slug: "cats",
+      },
+      {
+        description: "what books are made of",
+        slug: "paper",
+      },
+    ];
+
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toEqual(topics);
+      });
+  });
+  test("should return 404 for invalid request", () => {
+    return request(app).get("/api/topics/1").expect(404);
+  });
+});
```

Comment:
Where in your code have you implemented an error handler to send back a 404 status, and what other info should we send back to a user to accompany this status code?